pkgs/development/compilers/lobster/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+
+# Linux deps
+, libGL
+, xorg
+
+# Darwin deps
+, cf-private
+, Cocoa
+, AudioToolbox
+, OpenGL
+, Foundation
+, ForceFeedback
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lobster";
+  version = "unstable-2020-01-22";
+
+  src = fetchFromGitHub {
+    owner = "aardappel";
+    repo = pname;
+    rev = "42aca241f3244d159ef1f0b336f4649954313c15";
+    sha256 = "190c2f3sq9vryx7lpi9xmnf6jb2fiy0jrlgz1arkak0f0jxbz4sx";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = if stdenv.isDarwin
+    then [
+      cf-private
+      Cocoa
+      AudioToolbox
+      OpenGL
+      Foundation
+      ForceFeedback
+    ]
+    else [
+      libGL
+      xorg.libX11
+      xorg.libXext
+    ];
+
+  preConfigure = "cd dev";
+  enableParallelBuilding = true;
+
+  meta = with stdenv.lib; {
+    homepage = "http://strlen.com/lobster";
```

Comment:
this is https https://aardappel.github.io/lobster/README_FIRST.html, but links to this site too

Reply:
Are you saying that I should use that url for `homepage`? Shouldn't it be the canonical one?